### ISSUE TITLE
[Request] Send command to all clients.

### ISSUE THREAD

**Issue by ArlenRicard** [xp4xbox/Python-Backdoor#7]
Another possibility that if implemented, would save the day of many people.
If there is a way to send a command to multiple hosts at once.
For example:
On 200 hosts I have a scheduled task called update. And I use the command _schtasks /run /tn "update"_
However, I need to log in one by one to trigger this command.
If you had the possibility to add an extra command _--SS send to all hosts_.
If you had this call sent using CMD, you could use several other commands that are common to all hosts on the network.

The python-backdoor project is a giant project in my opinion. There are only a few implementations left so we can interact with multiple hosts at once, which run the same scripts for everyone, and lessen the task of one-by-one access.

**Comment by xp4xbox** [comment]
Its actually a very small project compared to others like https://github.com/n1nj4sec/pupy.

**Comment by xp4xbox** [comment]
Try the latest commit, I added this feature plus a few minor bug fixes. To use this feature simple run `--s command` and it will run on all users. There will also be a log file generated called `command_log` which will return the log information for every client. I added this since I feel like it would help a lot for especially system admins.

**Comment by xp4xbox** [comment]
I just tested it out with multiple users, works perfectly. So I will close this.

**Comment by ArlenRicard** [comment]
Wonderful!. I just tested here, it worked perfectly. The log file was well thought out too.
That's it ... The system is getting more and more complete.
There's another easteregg I've set up here, similar to yours `--g`. from the project.
Implemented using tight VNC. I press a key, it installs and configures the tight, and already makes the reverse connection in my server that is in listen mode.
The difference is that I did everything in batch. As I'm still getting the hang of python, but if you want I can send it here and you convert it to python language if you find it interesting.

But thanks, is already helping and much.
